core/base/db-upgrades-manager.php

```diff
@@ -86,7 +86,7 @@ public function admin_notice_start_upgrade() {
 	public function admin_notice_upgrade_is_running() {
 		$upgrade_link = $this->get_continue_action_url();
 		$message = '<p>' . sprintf( __( '%s Database update process is running in the background.', 'elementor' ), $this->get_updater_label() ) . '</p>';
-		$message .= '<p>' . sprintf( 'Taking a while? <a href="%s" class="button-primary">Click here to run it now</a>', $upgrade_link ) . '</p>';
+		$message .= '<p>' . __( '%s Database update process is running in the background.', 'elementor' ) . '<a href="' . $upgrade_link . '" class="button-primary">' . __( 'Click here to run it now', 'elementor' ) . '</a></p>';
```

Comment:
It looks like the text `Taking a while?`... is missing. and the text from line 88 `Database update process` is duplicated.

Reply:
Copy paste from the wrong line...

In any case it was fixed.